src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedTryDepthCheck.java

```diff
@@ -48,7 +86,39 @@
  *   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Example of violation:</p>
+ * <pre>
+ * try {
+ *     try {
+ *         try {
+ *             try {
+ *                 try { // violation, current depth is 4, max allowed is 3
+ *                 } catch (Exception e) {
+ *                 }
+ *             } catch (Exception e) {
+ *             }
+ *         } catch (Exception e) {
+ *         }
+ *     } catch (Exception e) {
+ *     }
+ * } catch (Exception e) {
+ * }
+ *</pre>
+ *<p>Example of correct code:</p>
+ *<pre>
+ * try {
+ *     try {
+ *         try {
+ *             try { // OK
```

Comment:
`// OK, current depth is 3, max allowed depth is 3`

Reply:
Corrected, Done